demo/train.py

```diff
@@ -32,11 +34,11 @@ def saveModel(esHost, scriptName, featureSet, modelFname):
     with open(modelFname) as modelFile:
         modelContent = modelFile.read()
         path = "_ltr/_featureset/%s/_createmodel" % featureSet
-        fullPath = urljoin(esHost, path)
+        fullPath = urljoin(ES_HOST, path)
```

Comment:
we reference ES_HOST here, but it's only imported to the `__name__ == "__main__"` block below. the import should probably be moved to top of file.